<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="newGame"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="Music"/>
    <addaction name="separator"/>
    <addaction name="nextLevel"/>
    <addaction name="lastLevel"/>
    <addaction name="noWall"/>
    <addaction name="showPath"/>
    <addaction name="viewOpen"/>
    <addaction name="showTrap"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="Help"/>
    <addaction name="About"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newGame">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="nextLevel">
   <property name="text">
    <string>Next Level</string>
   </property>
  </action>
  <action name="lastLevel">
   <property name="text">
    <string>Previous Level</string>
   </property>
  </action>
  <action name="noWall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore Wall</string>
   </property>
  </action>
  <action name="Music">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Music On/Off</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="showPath">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Path</string>
   </property>
  </action>
  <action name="viewOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Visible</string>
   </property>
  </action>
  <action name="showTrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Traps</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
